Server/VikingVault.Services/TransactionService.cs

```diff
@@ -19,7 +21,19 @@ public TransactionService(VikingVaultDbContext context)
 
         public Transaction AddTransaction(Transaction tran)
```

Comment:
Don't use abbreviations for names. Rename it to transaction

Reply:
Done! Thanks!